Studio-Lovelies/GG-JointJustice-Unity
# Feature/missing assets and animations

## Summary

Adds the following missing assets:
- Character profile icons
- Ross' Evil Laugh sound effect
- BoyBoy animation sprites + associated animation
- Main menu logo

Also fixes Ross' breakdown animation to make him not be visible on the final frame, and adds character profile icons to their associated ActorData object

## Testing instructions

Make sure tests run, and make sure the sprites have been added to the correct folders, and correct animations/scriptable objects/scenes where applicable

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #144 
closes #133 
closes #124 
closes #107 
closes #122
